### PR TITLE
Create blog page

### PR DESCRIPTION
**Pull request by wali39**
From PSD:
blog page2

By Coding:
blogs3

 ### `Blog page  are  fully responsive please check it`

### CODE REVIEW

**Comment by farhan2077** [comment]
Bottom section after image needs more padding in left, right and bottom side

**Comment by wali39** [comment]
> Bottom section after image needs more padding in left, right and bottom side

fixed

**Comment by farhan2077** [comment]
> > Bottom section after image needs more padding in left, right and bottom side
> 
> fixed

Great!

File: src/components/blogs/Blogs.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  },
+];
+
+const date = new Date().toDateString();
+
+export default function Blogs() {
+  return (
+    <div className={styles.container}>
+      <Row gutter={[8, 8]} justify="center">
+        {titleAndDescription.map((cardDetails) => (
```
Comment (alamiiin) [inline src/components/blogs/Blogs.js]: `cardDetails` should be replaced with `blog`

File: src/components/blogs/BreadCrump.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import { Pagination } from "antd";
+import styles from "./Blogs.module.css";
+
+export default function BreadCrump() {
```
Comment (alamiiin) [inline src/components/blogs/BreadCrump.js]: fix typo `Breadcrumb`

**Review by alamiiin — APPROVED** [review]

**Pull request merged**